packages/shared/src/components/auth/AuthModal.tsx

```diff
@@ -72,6 +72,7 @@ export default function AuthModal({
         trigger={trigger}
         isLoginFlow={isLogoutFlow || loginState?.isLogin}
         defaultDisplay={defaultDisplay}
+        targetId={isOnboardingV4dot5 ? OnboardingV4dot5.V4dot5 : undefined}
```

Comment:
Does this not need a fallback to v4 instead of undefined? (not entirely sure how it's used further)

Reply:
Well, the `AuthDisplay.OnboardingSignup` is also used in the AuthenticationBanner, but that doesn't have targetId set on AuthOptions

Reply:
Hmm yeah to many mixins going on.
I'm afraid we are missing some tracking everywhere.

Reply:
Yeah, definitely possible that something is lost somewhere somehow.